source/plugins/philips/devices/colorLight/Configuration

```diff
@@ -3,9 +3,9 @@
   "class": "ColorLight",
   "type" : "Light",
   "required_info": {
-    "bridge_ip": "127.0.0.1",
-    "search_method": "reachable",
-    "user": "unknown"
+    "bridge_ip": "The Ip address of the bridge",
+    "search_method": "The search method for finding the light (default: Reachable)",
```

Comment:
Shouldn't this "default: Reachable" be without capital like "default: reachable" (as it was in the old version, for clarity)